frontend/app/_components/client-dashboard-components/ClientStatus.js

```diff
@@ -0,0 +1,53 @@
+"use client";
+import { Doughnut } from "react-chartjs-2";
+import {
+  Chart,
+  ArcElement,
+  DoughnutController,
+} from "chart.js";
+import { FlagIcon } from "@heroicons/react/24/solid";
+
+Chart.register(ArcElement, DoughnutController);
+
+function ClientStatus({ statusCounts }) {
+  // Destructure counts from the statusCounts prop
+  const { completed, inProgress, pending } = statusCounts;
+
+  // Dynamic data for the Doughnut chart based on status counts
+  const data = {
+    labels: ["Completed", "In Progress", "Pending"],
+    datasets: [
```

Comment:
I can see you have the datasets here, But it seems like you never use it? So what the purpose to create datasets here? Is that for styles or something?

Reply:
I used it for status clients